src/main/java/org/folio/circulation/domain/UpdateItem.java

```diff
@@ -155,11 +155,18 @@ public Item onDestinationServicePointUpdate(Item item, ServicePoint servicePoint
   private ItemStatus itemStatusOnRequestCreation(
     RequestAndRelatedRecords requestAndRelatedRecords) {
 
-    RequestQueue requestQueue = requestAndRelatedRecords.getRequestQueue();
+    RequestType type = requestAndRelatedRecords.getRequest().getRequestType();
 
-    return requestQueue.hasOutstandingRequests()
-      ? requestQueue.getHighestPriorityRequest().checkedOutItemStatus()
-      : requestAndRelatedRecords.getRequest().checkedOutItemStatus();
+    if (type == RequestType.PAGE){
+      return ItemStatus.PAGED;
+    } else {
+      //leave existing logic the same
+      RequestQueue requestQueue = requestAndRelatedRecords.getRequestQueue();
+
+      return requestQueue.hasOutstandingRequests()
+        ? requestQueue.getHighestPriorityRequest().checkedOutItemStatus()
+        : requestAndRelatedRecords.getRequest().checkedOutItemStatus();
```

Comment:
Given that we have implemented the entire white list for all request types. I think this code will lead to some unexpected behaviour. 

Recall and hold request types will try to set the status to `Checked out` (as this was the only allowed status before the change).

I think we have three options
* Remove this code entirely, which will likely result in the correct behaviour, (however will not really be checked by tests except for `Checked out`)
* Temporarily change the whitelist for hold and recall requests to include `Checked out` (to make this consistent, if limited)
* Leave it as is, and accept that it will be resolved during CIRC-207 and CIRC-208

What do you think?

Reply:
I think we better leave it as is and resolve it in CIRC-207, CIRC-208, rather than introducing technical debt (option 2) or risks (option 1).

Reply:
Ok